.github/workflows/ci.yml

```diff
@@ -2,11 +2,7 @@ name: CI
 
 on:
   push:
-    paths:
-      - src/**
-      - tests/**
-      - scripts/**
-      - .github/workflows/ci.yml
+    branches: ["**"]
```

Comment:
What is the point of adding this? Does it do something different from the default?

Reply:
yes, the workflow doesn't run on tag push when `branches` is specified